src/language/api/verificationDiagnostics.ts

```diff
@@ -0,0 +1,85 @@
+import { Position } from 'vscode';
+import { Diagnostic, DocumentUri, Range, integer } from 'vscode-languageclient';
+
+export interface IVerificationDiagnosticsParams {
+  uri: DocumentUri;
+  version?: integer;
+  diagnostics: Diagnostic[];
+  perNodeDiagnostic: NodeDiagnostic[];
+  diagnosticsAreResolutionErrors: boolean;
+  perLineDiagnostic: LineVerificationStatus[];
+}
+
+export interface NodeDiagnostic {
```

Comment:
```suggestion
export interface INodeDiagnostic {
```

Prepend with `I`.

Reply:
We are also discussing the name "NodeDiagnostic" here
https://github.com/dafny-lang/dafny/pull/1942#discussion_r838571564
Meanwhile, I'll prefix it with an I